_posts/en/posts/2016-01-25-segwit-benefits.md

```diff
@@ -0,0 +1,328 @@
+Segwit resolves this by including a version number for scripts, so that
+additional opcodes that would have required a hard-fork to be used
+in non-segwit transactions can instead be supported by simply increasing the
+script version.
+
+### Who benefits?
+
+Easier changes to script opcodes will make advanced scripting in Bitcoin
+easier. This includes changes such as introducing Schnorr signatures,
+using key recovery to shrink signature sizes, supporting sidechains,
+or creating even smarter contracts by using Merklized Abstract Syntax
+Trees (MAST) and other research-level ideas.
+
+## Reducing UTXO growth
+
+The Unspent Transaction Output (UTXO) database is maintained by each
+validating Bitcoin node in order to determine whether new transactions are
+valid or fraudulent. For efficient operation of the network, this database
+needs to be very quick to query and modify, and should ideally be able
+to fit in main memory (RAM), so keeping the database's
+byte size as small as possible is valuable.
+
+This becomes more difficult as Bitcoin grows, as each new user must have
+at least one UTXO entry of their own and will prefer having multiple
+entries so that they can use techniques that help improve their privacy
+and flexibility, as well as to use techniques such as Lightning that
+don't require putting all of their data on the block chain.
+
+Segwit improves the situation here by providing a 75% byte-size discount
+to people who create their transactions in a way that reduces the amount
+of UTXO space they use so that we store a greater number of UTXO entries
+in each megabyte of data.
+
+### Who benefits?
+
+Reduced UTXO growth will benefit miners, businesses, and users who run full nodes,
+which in turn helps maintain the current security of the Bitcoin network
+as more users enter the system.
+
+## Compact fraud proofs
+
+As the Bitcoin userbase expands, validating the entire blockchain
+naturally becomes more expensive. To maintain the decentralised, trustless
+nature of Bitcoin, it is important to allow those who cannot afford to
+validate the entire blockchain to at least be able to cheaply validate
+as much of it as they can afford.
+
+Segwit improves the situation here by allowing a future soft-fork to
+extend the witness structure to include commitment data, which will allow lightweight (SPV)
+clients to enforce consensus rules such such as the number of bitcoins
+introduced in a block, the size of a block, and the number of sigops
+used in a block.
+
+### Who benefits?
+
+Fraud proofs allow SPV users to help enforce Bitcoin's consensus rules,
+which will potentially greatly increase the security of the Bitcoin
+network as a whole, as well as reduce the ways in which individual users
+can be attacked.
+
+These fraud proofs can be added to the witness data structure as part
+of a future soft-fork, and they'll help SPV clients enforce the rules
+even on transactions that don't make use of the segwit features.
+
+## Discarding signatures
```

Comment:
"Skipping signatures becomes possible" maybe?

IMO what doesn't sit right with "skipping" is that it suggests a loss of security - but that is in fact the case, so it isn't a problem.